packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -81,6 +81,67 @@ enum SystemUiOverlay {
   bottom,
 }
 
+/// Signature for listening to changes in the [SystemUiMode].
+///
+/// Used by [SystemChrome.setEnabledSystemUIMode].
+typedef SystemUiChangeCallback = void Function(bool systemOverlaysAreVisible);
+
+/// Specifies Android fullscreen mode.
+///
+/// Used by [SystemChrome.setEnabledSystemUIMode].
+enum SystemUiMode {
+  /// Fullscreen display without status or navigation bars.
+  ///
+  /// Available starting at SDK 16 or Android J. Tapping on the screen displays
```

Comment:
What happens if you specify this on devices running an older version of Android (or don't run android at all, e.g. iOS)?